Camera/schema.json

```diff
@@ -43,7 +43,7 @@
             "DOME",
             "DAY/NIGHT",
             "C-MOUNT",
-            "BULLET"
+            "BULLET"         
```

Comment:
Need TRAFFIC option in the enumeration

Reply:
hmm initially I considered extending this, but this is not our goal. We just want a description of its usage to be for Traffic, so we think its best if it goes with the enum in cameraUsage. But it looks as if this is a free enumerator.

Reply:
ok, then please remove it from the description

Reply:
I am not seeing traffic removed. Am I wrong?